contents/docs/advanced/content-security-policy.md

```diff
@@ -51,6 +51,22 @@ Depending on your compliance needs you can either:
 
 **NOTE**: This list should be enough at the time of writing. As the PostHog application changes rapidly, it is possible that other directives may be needed over time for loading the Toolbar. If you experience issues after implementing one of the above solutions, you can typically debug in the browser tools which part of the CSP is blocking requests.
 
+## Supporting none directives
```

Comment:
I'd love for us to have a way to clarify from which `posthog-js` version onwards this setting exists - same applies to other documentation. I will come up with something tomorrow when Im back in town :)

Reply:
Yeah - this might just get out of hand really quick though